Update list with new entries

svetlik opened the pull request:
Updating the list with a lot more entries. The bigger list had the domain of our last attack.

As the list has 110k+ entries, would it pose any issues? Would it make sense to keep it somewhere else? 

The source for the list can be found [here](https://github.com/ivolo/disposable-email-domains).

svetlik committed:
Update list with new entries

wesleyeewong commented:
How many new ones did you add? Can you make the diff smaller? IE don't change `'` => `"`

motionless commented:
How did you generate the new entries?
Is this an update from the fork or is this from our own data?

We might be able to shorten the list because of the MX lookup. 
Maybe a quick call would make sense?

svetlik commented:
Based on the call with @motionless will update the list by:
- checking for duplicates
- checking if any of the entries is missing an mx record, in which case will filter those out.
@wesleyeewong I will then include the result and switch back to single quotes.

svetlik committed:
Filter out entries without mx record and merge with existing list

Merging previous list with entries from the new list that were confirmed
to have an mx record.

svetlik committed:
Fix indentation
